projects/demo/src/modules/pipes/is-present/examples/1/component.ts

```diff
@@ -0,0 +1,32 @@
+import {Component} from '@angular/core';
+import {FormControl} from '@angular/forms';
+import {changeDetection} from '@demo/emulate/change-detection';
+import {encapsulation} from '@demo/emulate/encapsulation';
+
+@Component({
+    selector: `tui-is-present-example1`,
+    templateUrl: `./template.html`,
+    styleUrls: [`./style.less`],
+    changeDetection,
+    encapsulation,
+})
+export class TuiIsPresentExample1 {
+    readonly items = [`String`, `null`, `undefined`];
+    readonly control = new FormControl(null);
+
+    get value(): string | null | undefined {
+        return this.objectifyValue(this.control.value ?? ``);
+    }
+
+    private objectifyValue(value: string): string | null | undefined {
+        if (value === `null`) {
```

Comment:
nit: Looks like a job for `switch` :)

Reply:
do you want to keep this example or completely replace it with the one above?

Reply:
I went ahead and replaced it since the new example is more useful 🙂

Reply:
@waterplea please check if it's not too complicated

Reply:
Looks great. Thank you!